src/Discovery/Discovery.tsx

```diff
@@ -265,7 +228,8 @@ const Discovery: React.FunctionComponent<DiscoveryBetaProps> = (props: Discovery
       }],
       onFilter: (value, record) => record[accessibleFieldName] === value,
       ellipsis: false,
-      width: undefined,
+      width: '106px',
```

Comment:
this is a good point that you make. but i'll push back and say -- there are plenty of hardcoded, non-configurable pixel widths in all of our CSS files. some hardcoding is necessary to present a standardized UI, including for responsive UIs. in this case, i hardcoded a column width for one column that is unlikely to shift so that other columns are allowed to shift freely (this is a limitation of the antd table library; some columns must have fixed widths https://ant.design/components/table/#components-table-demo-fixed-header ). if a use case arises that requires configurability of this value, making it configurable will be straightforward. but for now, i think this hardcoded value is appropriate in this context